Update integration libraries (#13277)
PrefectHQ/prefect: docs/integrations/prefect-gcp/index.md

    bigquery_client = bigquery.Client(credentials=google_auth_credentials)
```

> If you want to access the underlying client, `prefect-gcp` exposes a `get_client` method from `GcpCredentials`.

Reviewer: ```suggestion
To access the underlying client, use the `get_client` method from `GcpCredentials`.
```
